### PR TITLE
Add https info to httpd

### PR DESCRIPTION
**Pull request by yosifkit**
docs for https://github.com/docker-library/httpd/pull/2
ping @tianon

### CODE REVIEW

File: httpd/content.md [hunk 1]
```diff
@@ -44,3 +44,18 @@ configuration in as `/usr/local/apache2/conf/httpd.conf`.
 
     FROM httpd:2.4
     COPY ./my-httpd.conf /usr/local/apache2/conf/httpd.conf
+
+#### SSL/HTTPS
+
+If you want to run your web traffic over SSL, the simplest setup is to `ADD` or
```
Comment (tianon) [inline httpd/content.md]: `s/ADD/COPY/`

File: httpd/content.md [hunk 2]
```diff
@@ -44,3 +44,18 @@ configuration in as `/usr/local/apache2/conf/httpd.conf`.
...
+mount (`-v`) your `server.crt` and `server.key` into `/usr/local/apache2/conf/`
+and then customize the `/usr/local/apache2/conf/httpd.conf` by removing the
+comment from the line with `#Include conf/extra/httpd-ssl.conf`.  This config
+file will use the certificate files previously added and tell the daemon to also
+listen on port 443. Be sure to also add `-p 443:443` to your `docker run` to
```
Comment (tianon) [inline httpd/content.md]: `also add ...` -> `also add something like ...`

**Comment by tianon** [comment]
two minor nits, but otherwise LGTM

**Comment by Moghedrin** [comment]
LGTM

**Comment by yosifkit** [comment]
Fixed, @tianon

**Comment by tianon** [comment]
LGTM

**Pull request merged**